app/src/main/res/layout/activity_main.xml

```diff
@@ -27,27 +32,36 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_weight="1"
-            android:spinnerMode="dropdown"/>
+            android:spinnerMode="dropdown" />
 
         <com.toptoche.searchablespinnerlibrary.SearchableSpinner
             android:id="@+id/to_station"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_weight="1"/>
+            android:layout_weight="1" />
 
         <Button
             android:id="@+id/search_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Search"/>
-
+            android:text="Search" />
```

Comment:
Could we pull this out into the strings.xml file please, as well as any other strings. Could we also pull styles out into the styles.xml, e.g. you could have a 'title' style which sets text alignment, colour and style, will keep xml files reusable, keep styles consistent and will also mean you only need to make style changes in one place if e.g. the colour you want to use everywhere changes